src/screens/Profile.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import { withStore } from "../assets/helpers/store";
+import { disableBodyScroll, enableBodyScroll, clearAllBodyScrollLocks } from 'body-scroll-lock';
+import Modal from "../components/Modals/LoginSignUpRedirectModal";
+
+class Profile extends Component {
+   constructor(props) {
+      super(props);
+      this.state = {
+         viewOnly: true,
+         selectedCategory: '',
+         events: [],
+         loggedInUserName: 'User',
+         userLoggedIn: 'false',
+         loggedInUser: {},
+         tggle: false,
+      }
```

Comment:
for loggedInUserName, userLoggedIn, loggedInUser etc. we can just set them to 'userLoggedIn: this.props.store.loggedInUser` etc.

Reply:
This way, we don't have to set the state in componentDidMount, it will already be set and we can handle modal showing etc. in there only